src/store/root-store.js

```diff
@@ -0,0 +1,13 @@
+import UserStore from "./user-store";
+import PlaylistStore from "./playlist-store";
+
+class RootStore {
```

Comment:
so the root store is a bit of a confusing topic. don't worry too much about this for now, we'll go over it more later

but just know that to use the stores in our components, we should be importing from the root store instead of directly importing `user-store` or `playlist-store`.